src/SourceGenerators/Uno.UI.SourceGenerators/XamlGenerator/XamlFileGenerator.cs

```diff
@@ -5326,8 +5331,8 @@ private void TrySetParsing(IIndentedStringBuilder writer, XamlObjectDefinition o
 			TryAnnotateWithGeneratorSource(writer);
 			if (HasIsParsing(objectDefinition.Type))
 			{
-				writer.AppendLineInvariant("IsParsing = true");
-				writer.AppendLineInvariant(isInitializer ? "," : ";");
+				// Append [base.]IsParsing = true(, or ;)
+				writer.AppendLineInvariant($"{(!isInitializer ? "base." : string.Empty)}IsParsing = true{(isInitializer ? "," : ";")}");
```

Comment:
Let's decompose this for readability:

```suggestion
                var basePrefix = !isInitializer ? "base." : string.Empty;
                var initializerSuffix = isInitializer ? "," : ";";
				writer.AppendLineInvariant($"{basePrefix }IsParsing = true{initializerSuffix}");
```